fortysecondscv.cls

```diff
@@ -135,6 +135,12 @@
 	\AtBeginDocument{\apptocmd{\personaldata}{\hline}{}{}}
 }
 
+% put name and job after (default) or before picture
+\newtoggle{tnameabovepicture}
+\togglefalse{tnameabovepicture}
+\DeclareOptionX{nameabovepicture}{}
```

Comment:
This option currently has no effect because of improper bracket placement  :sweat_smile:.

The closing bracket needs to go in L143 and a `%` has to be added at the EOL like so

```
\DeclareOptionX{nameabovepicture}{%
	\toggletrue{tnameabovepicture}
}
```

Reply:
Good catch. I seem to have rushed the commit a little. Sorry about that.

Since then, I actually had a need to completely hide the name and job title around the profile picture. So I changed the option so it can say 'after' / 'before' / 'hidden' (or anything else than 'after' and 'before').